src/schema.ts

```diff
@@ -177,6 +177,7 @@ export const typeDefs = gql`
       filterIds: String
       subjectId: String
       removeRelatedContent: String
```

Comment:
Denne parameteren blei lagt til i graphqlforholdsvis nylig og brukes kun fra ndla-frontend. Den kan nok fjernes fra schema.

Reply:
`removeRelatedContent`?

Reply:
Ja. Den blei innført for å gjøre det likt som i stier.